js/components/scrubberBar.jsx

```diff
@@ -263,6 +263,10 @@ class ScrubberBar extends React.Component {
         : event.nativeEvent.offsetX;
     }
 
+    if (event.target.className.match('oo-marker')) {
```

Comment:
Do we need to test it?

Reply:
I tried, but looking at the test scenarios of the scrubber bar I didn't find any elaborate test over the scrubber bar actions ( mouseDown, mouseUp)...the ones that are written only check the existence of the event and nothing more.

Reply:
what if `event.target` is a child of `.oo-marker`?

Reply:
`.oo-marker` it's not intended to have child elements, so in this case we will never get the event propagation from a different source.